package.json

```diff
@@ -4,17 +4,52 @@
   "scripts": {
     "build": "node ./scripts/build.js",
     "clean": "git clean -fdX .",
-    "postinstall": "node ./scripts/postinstall.js",
+    "postinstall": "lerna run prepare",
```

Comment:
Instead of calling `lerna bootstrap`, run each package's `prepare script. All of the `prepare` scripts (besides the website's) call `npm run build`.